packages/cozy-scripts/config/webpack.target.mobile.js

```diff
@@ -7,7 +7,7 @@ const webpack = require('webpack')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin')
 
-const { production, isDebugMode } = require('./webpack.vars')
+const { environment, isDebugMode } = require('./webpack.vars')
 const manifest = fs.readJsonSync(paths.appManifest())
 
```

Comment:
Actually, you want to have the `production` variable here like

```suggestion
 const production = environment === 'production
```